helper/identity/types.proto

```diff
@@ -110,6 +110,10 @@ message Entity {
 	// MFASecrets holds the MFA secrets indexed by the identifier of the MFA
 	// method configuration.
 	//map<string, mfa.Secret> mfa_secrets = 10;
```

Comment:
I’m suspicious having this commented out might break if a user upgrades from oss -> ent